pkg/plugin/plugin.go

```diff
@@ -2,13 +2,13 @@ package plugin
 
 // Plugin defines an external plugin and its associated data.
```

Comment:
Adding that we have toml keys for input and json keys for output could be useful.

Reply:
I've added some notes about it.